jetbrains-gateway/main.tf

```
@@ -46,6 +50,22 @@ variable "coder_parameter_order" {
   default     = null
 }
 
+variable "latest" {
+  type        = bool
+  description = "Whether to fetch the latest version of the IDE."
+  default     = true
```

```suggestion
  default     = false
```
@mafredri, should we default `latest` to `false` so that the existing behavior to use the hardcoded version is retained?

---

It’s certainly less risky, so I’m all for it 👍